server/src/controllers/buttonController.js

```diff
@@ -108,4 +105,124 @@ const saveJob = (req, res, data) => {
     ////////////////////////////////////////////////////////////////////
 }
 
+const locationButtons = (req, res, data) => {
+  let id = data.user.id;
+  let teamId = data.team.id;
+
+  User.find({
+    where: { slackUserId: data.user.id }
+  })
+  .then(user => {
+    console.log(user.dataValues);
+    let location = user.dataValues.location;
+    let reply;
+    //if button clicked was view, send a direct message 
+    //to the user with user location, and change the button to clicked!
+    if (data.actions[0].value === 'view') {
+      reply = {
+        type: 'message',
+        text: data.original_message.text,
+        attachments: [
+          {
+            text: `Your current search loation is:\n ${location}`,
+            callback_id: 'something else',
+            color: `#3AA3E3`,
+            attachment_type: `default`
+          }
+        ]
+      };
+      res.json(reply);
+    } else {
+      //if the button clicked was update, update the location
+      //and notify user the location was updated!
+      reply = buttonUpdater(data, "Update", 1, 'primary', 'something else');
+      reply.attachments[0].text = 'Respond to Buckleybot to update search location',
```

Comment:
at some point we need to become consistent with the nomenclature of the bot. Right now we are using "Buckley", "BuckleyBot", and "Buckleybot" interchangeably